hi-ml-azure/src/health_azure/datasets.py

```diff
@@ -317,6 +317,7 @@ def __init__(
         use_mounting: Optional[bool] = None,
         target_folder: Optional[PathOrString] = None,
         local_folder: Optional[PathOrString] = None,
+        data_name: Optional[str] = None,
```

Comment:
I agree this is not great. `data_asset_name` is not good because that's what `name` refers to here. I've just used `alias` in the path. I like `data_asset_identifier`, although it's maybe a bit long.

Can you give a final veredict?

Reply:
@ant0nsc shall we keep `data_name` and improve the docs? `data_asset_identifier` might also not be good because outputs are not necessarily data assets.

Reply:
I've gone for `data_name` for consistency and added examples in the docstring and the `datasets` Markdown file.